README.md

```diff
@@ -35,9 +34,9 @@ To see more options of helm values [here](helm/README.md)
 * docker
 * [operator-sdk v0.13.0](https://github.com/operator-framework/operator-sdk/releases/tag/v0.13.0)
 * make
-* kubectl v1.14+(< v1.19)
```

Comment:
kubectl can work over multiple versions, but best case to not drift more than 2 minor versions from your cluster.

(minor)

Reply:
client kubectl 1.20 works. I am gonna update docs with (< v1.21)